src/main/java/com/redhat/jenkins/plugins/ci/pipeline/CIMessageSubscriberStep.java

```diff
@@ -182,8 +172,6 @@ public void run() {
         public void stop(@Nonnull Throwable cause) throws Exception {
             if (task != null) {
                 getContext().get(TaskListener.class).getLogger().println("in stop of watcher");
-                watcher.interrupt();
```

Comment:
not sure you ever responded to my question as to why this was removed...

Reply:
That line wasn't flagged with the comment.

Reply:
I removed the watcher completely.  I'm not sure why it was added instead of just calling CIMessageSubscriberBuilder.waitForMessage.

The watcher code seems to just duplicate that code.